app/views/_user_util_links.html.erb

```diff
@@ -24,7 +24,7 @@
   <% else %>
     <li>
       <%= link_to main_app.new_user_session_path do %>
-        <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span > t("sufia.toolbar.profile.login")
+        <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span > <%= t("sufia.toolbar.profile.login") %>
```

Comment:
Yikes. Surprised our test suite didn't flag this! Thanks for the assist, @jcoyne.